uxarray/grid/grid.py

```diff
@@ -198,6 +200,28 @@ def from_face_vertices(cls,
 
         return cls(grid_ds, source_grid_spec="Face Vertices")
 
+    def validate(self):
+        """Validate a grid object check for common errors, such as:
+
+            - Duplicate nodes
+            - Connectivity
+            - Face areas (non zero)
+        Raises
+        ------
+        RuntimeError
+            If unsupported grid type provided
+        """
+        # If the mesh file is loaded correctly, we have the underlying file format as UGRID
+        # Test if the file is a valid ugrid file format or not
+        print("Validating the mesh...")
+
+        # call the check_connectivity and check_duplicate_nodes functions from validation.py
+        _check_duplicate_nodes(self)
+        _check_connectivity(self)
+        _check_area(self)
+
+        return True
```

Comment:
Does this return statement make sense in the context of the above functions? It should only return `True` if all three of the above helpers return true, right?